### PR TITLE
release: validate changelog in tests

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: src/scripts/changelog.zig [hunk 1]
```diff
@@ -9,6 +9,8 @@ const ReleaseTriple = @import("../multiversioning.zig").ReleaseTriple;
 
 const log = std.log;
 
+const const_max_bytes = 10 * 1024 * 1024;
```
Comment (sentientwaffle) [inline src/scripts/changelog.zig]: `const_bytes_max`?

Comment (sentientwaffle) [reply src/scripts/changelog.zig]: or `const_size_max`?

Comment (sentientwaffle) [reply src/scripts/changelog.zig]: Wait, why is `const_` the prefix?

Comment (matklad) [reply src/scripts/changelog.zig]: :fearful: not sure what happened here, definitely should be `changelog_bytes_max`

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**